### DISCUSSION TITLE
Access denied when client tries to listen to music (mp3/wav) file - Cyberduck

### DISCUSSION THREAD

**Discussion by alineb07** [iterate-ch/cyberduck#16496] [Q&A]
Cyberduck support: Hi guys, the client receives this Access denied message (video attached) when he tries just to listen to the music file that on the program. I will be grateful for your opinion and help what is the problem.

https://github.com/user-attachments/assets/3158ba06-6a91-4930-8aa5-9e08a3c7bfa8

**Comment by dkocher** [comment]
Can you reproduce if you disable [segmented downloads](https://docs.cyberduck.io/cyberduck/download/#segmented-downloads) in _ Preferences → Transfers → General → Downloads → Segmented download with multiple connections for file_?
